<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadGame</class>
 <widget class="QWidget" name="LoadGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>存档管理</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_LoadGame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>读取存档</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>删除存档</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_home">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>回到主界面</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
